modules/openideal_user/openideal_user.install

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the openideal user module.
+ */
+
+use Drupal\block_content\Entity\BlockContent;
+
+/**
+ * Implements hook_update_N().
+ */
+function openideal_user_update_8001(&$sandbox) {
+  $blocks = \Drupal::entityTypeManager()->getStorage('block_content')->loadByProperties(['info' => 'login']);
+  if (!empty($blocks)) {
+    return;
+  }
+
+  // Create a block_content for right section of login screen.
+  $module_handler = \Drupal::service('module_handler');
```

Comment:
So there is no way to import a single block from default_content content storage?

Reply:
I moved the whole logic related to that block and user.login page into the openideal_login module, please check it out.